Wrappers/Python/test/test_Gradient.py

```diff
@@ -73,7 +73,7 @@ def test_GradientOperator_linearity(self):
                                                     backend = backend, 
                                                     correlation = corr, method = method)
                             try:                                                    
-                                self.assertTrue(LinearOperator.dot_test(Grad, seed=5))
+                                self.assertTrue(LinearOperator.dot_test(Grad))
```

Comment:
I believe the `seed` should be passed. In the current implementation this has no effect as you removed it, but I'm against it ;)

Reply:
Fixed